bazel/README.md

```diff
@@ -409,8 +409,11 @@ bazel build -c dbg //test/common/http:async_client_impl_test
 gdb bazel-bin/test/common/http/async_client_impl_test
 ```
 
-Without the `-c dbg` Bazel option at the end of the command line the test
-binaries will not include debugging symbols and GDB will not be very useful.
+We need to use `-c dbg` Bazel option to generate debugging symbols and without
+that GDB will not be very useful. The debugging symbols are stored as separate
+debugging information files (.dwo files). When using RBE to build, it's
+necessary to add `--remote_download_outputs=all` to download those `.dwo` file
```

Comment:
Never recommend `--remote_download_outputs=all`, this is will be more build performance degradation than build without split dwarf, instead, just building `.dwp` target at same time would be enough.

Reply:
I see.

Reply:
I pushed the new version of README. Now it has included the commands to build `.dwp` file in the guide.